data.sql

```diff
@@ -44,3 +44,33 @@ UPDATE animals SET owner_id = (CASE
                                 WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
                              END);
 
+-- Join Tables Data
+
+-- Insert data into the vets table
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+-- Insert data into the specializations table
+INSERT INTO specializations (vet_id, species_id) VALUES
+(1, (SELECT id FROM species WHERE name = 'Pokemon')),
+(3, (SELECT id FROM species WHERE name = 'Digimon')),
+(3, (SELECT id FROM species WHERE name = 'Pokemon')),
+(4, (SELECT id FROM species WHERE name = 'Digimon'));
+
+-- Insert data into the visits table
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
+-- Provided data for visits
+(1, 1, '2020-05-24'),
+(1, 3, '2020-07-22'),
+(2, 4, '2021-02-02'),
+(3, 2, '2020-01-05'),
+(3, 2, '2020-03-08'),
+(3, 2, '2020-05-14'),
+(4, 3, '2021-05-04'),
+(5, 4, '2021-02-24'),
+(6, 3, '2020-05-24'),
+(6, 1, '2021-01-11'),
+(7, 2, '2022-05-14');
```

Comment:
👏 Good progress on the project! 👏 However, I noticed a couple of points that need attention:

- [ ] **Incomplete Entries in `visits` Table:** According to [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following%20data%20for%20visits,William%20Tatcher%20on%20Jan%2011th%2C%202021.), there should be 20 entries in the `visits` table, but currently, you've entered only 11. Please ensure that you follow the project requirements and add the remaining 9 entries to meet the specified criteria.

- [ ] **Verification of SQL Queries:** Since you've updated table data, I recommend running the new queries from the `queries.sql` file. Verify the output and, if necessary, update the screenshots in the PR Description to accurately reflect the changes made.

By addressing these points, you'll align your work with project requirements and provide a comprehensive overview in the PR. Keep up the good work! 🚀📊